app/views/posts/_expanded.html.erb

```diff
@@ -24,7 +24,7 @@
     </h1>
   <% end %>
 
-  <% if float_notice %>
+  <% if params[:float_notice] %>
```

Comment:
Does this work? This looks like it'd be looking for a parameter, not a variable. There are optional variables defined in a bunch of other views (including the post partial), if you want an example.

Reply:
It does work; I've tested it. I'm not sure what you mean by parameter vs variable.